.env.example

```diff
@@ -18,3 +18,7 @@ VUE_APP_ALIAS=
 VUE_APP_DEFAULT_LOG_LEVEL="error"
 VUE_APP_LOGIN_URL="http://launchpad.hotwax.io/login"
 VUE_APP_BROKER_JOB_ENUMS = {"REJ_ORDR":"JOB_BKR_REJ_ORD"}
+VUE_APP_GITBOOK_API_KEY="gb_api_hL97xDJyiI6FBhnWKdm8CP6N6Wjl5nZSjhK3Nyd7"
```

Comment:
should we be committing this to our repo?

Reply:
No, sir, we shouldn't. I have reverted the changes.